dom/src/index.ts

```diff
@@ -27,12 +27,16 @@ export {MainDOMSource} from './MainDOMSource';
  * https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener
  * about the `useCapture` and its purpose.
  * The other option is `preventDefault` that is set to false by default.
- * If set to true, the driver will automatically call `preventDefault()` on every event.
- *
- * `DOMSource.elements()` returns a stream of the DOM element(s) matched by the
- * selectors in the DOMSource. Also, `DOMSource.select(':root').elements()`
- * returns a stream of DOM element corresponding to the root (or container) of
- * the app on the DOM.
+ * If set to true, the driver will automatically call `preventDefault()` on
+ * every event.
+ *
+ * `DOMSource.elements()` returns a stream of arrays of the DOM elements matched
+ * by the selectors in the DOMSource. Also,
+ * `DOMSource.select(':root').elements()` returns a stream of arrays of the DOM
+ * element corresponding to the root (or container) of the app on the DOM.
+ * Special cases are `DOMSource.select('document').elements()` that returns a
+ * Stream of documents (no arrays) and `DOMSource.select('body').elements()`
+ * that returns a Stream of HTMLBodyElements (no arrays).
```

Comment:
I would mention the `element()` method.
Something like: "To get a single element there is an analog `element` function`

Reply:
Very good point, I'll write about that.